frontend/app_flowy/lib/workspace/presentation/widgets/menu/widget/top_bar.dart

```diff
@@ -4,24 +4,29 @@ import 'package:flowy_infra/image.dart';
 import 'package:flowy_infra_ui/style_widget/icon_button.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:provider/provider.dart';
```

Comment:
This line causes a warning, please fix

Reply:
I think the CI should block the codes with warnings?